### DISCUSSION TITLE
Adding getStaticProps to Index breaks / route

### DISCUSSION THREAD

**Discussion by AndrewSB** [vercel/next.js#13375] [Help]
Hey! I have a pretty simple static site built with v9.3.6, there's a problem I'm running into when adding 

```tsx
export async function getStaticProps() { ... }
```

to my `pages/index.tsx`.

Once deployed through Vercel, navigating to the URL's `/` route leads to a page with no javascript. The HTML sent down is just my `public/index.html` with the root div not populated.

Strangely enough, if I navigate to the deployment URL's `/index` path, I see the page with the statically fetched data.

I can't reproduce this locally - when going to `localhost:3000/` I see the index page correctly.
If I remove the `getStaticProps()` export, the deployment's `/` path begins to work correctly again.

Index.tsx looks like this

```tsx
type Props = {
  emptySearch?: SearchResultRow[]
}

function IndexPage(props?: Props) {
  ...
}

export async function getStaticProps() {
  ...
}

export default IndexPage
```

**Comment by lfades** [comment]
Hi @AndrewSB please provide a reproduction of the issue that we can take a look at, the current details you're providing aren't enough to replicate the issue.

**Reply by AndrewSB** [reply]
Hmm, I'm having issues coming up with a reduced example. I have a small patch (commenting out `getStaticProps` from the page)

I've added you as a collaborator to the project, the branch `fix-index` has the issue I'm working though 
`HEAD` comments out the function to get `/` to render (https://crazydata-cyruqsum9.now.sh)
`HEAD~1` is the state of master with `/` blank (https://crazydata-git-fix-index.andrewsb.now.sh)

**Reply by AndrewSB** [reply]
bump, any chance we can take a look at this soon?

**Reply by lfades** [reply]
Hi @AndrewSB, I didn't have the time to check out your branch, if possible please create a public reproduction.

> Hmm, I'm having issues coming up with a reduced example. I have a small patch (commenting out getStaticProps from the page)

This can be an indicator that there may be an strange library or config with unexpected behaviors 🤔

**Reply by AndrewSB** [reply]
totally understand, i'll try again to get a reduced repro
